indi-maxdomeii/maxdomeii.cpp

```diff
@@ -210,15 +209,16 @@ void MaxDomeII::TimerHit()
 
     // Watch dog
     nTimeSinceLastCommunication++;
-    if (WatchDogNP.np[0].value > 0 && WatchDogNP.np[0].value <= nTimeSinceLastCommunication)
```

Comment:
@knro , there are few lines which are using WatchDogNP.np[0].value.
Is it count? i.e I commented these lines. Or we simply using normal values/
Please let me know so I will update in this PR

Reply:
No, it's not count. It's the first element of this property. i.e. WatchDogNP[0].getValue()

Reply:
updated. Pushed a commit